### ISSUE TITLE
Trend error on staging sponsorship (send agreement)

### ISSUE THREAD

**Issue by hhlanglois** [louking/contracts#436]
I just did "Send Agreement" for a WDF sponsorship (Anderabi Management). It seems to have sent the agreement, but I got the following error:

```
Traceback (most recent call last):
  File "/var/www/[www.contracts.loutilities.com/venv/lib/python3.9/site-packages/loutilities/tables.py](http://www.contracts.loutilities.com/venv/lib/python3.9/site-packages/loutilities/tables.py)", line 733, in wrapped_f
    self.editor_method_posthook(request.form)
  File "/var/www/[www.contracts.loutilities.com/contracts/contracts/contracts/views/admin/sponsorcontract.py](http://www.contracts.loutilities.com/contracts/contracts/contracts/views/admin/sponsorcontract.py)", line 230, in editor_method_posthook
    calculateTrend(thesesponsorships)
  File "/var/www/[www.contracts.loutilities.com/contracts/contracts/contracts/trends.py](http://www.contracts.loutilities.com/contracts/contracts/contracts/trends.py)", line 36, in calculateTrend
    thisyear = int(thisyearsships[0].raceyear)
IndexError: list index out of range
```

**Comment by louking** [comment]
This looks like a test escape but I'm not sure of the case I missed.

Was the sponsorship was created first as tentative, then opened for Send Agreement?

If agreement was sent, I think it's safe to manually change the sponsorship state to `committed`

**Comment by hhlanglois** [comment]
The sponsorship was originally in the renewed-pending stage. I staged the
sponsorship to tentative when I sent the solicitation. When the sponsor
responded via email, I filled in a few more fields and hit the "send
agreement" button.

I manually set the state to committed, but it doesn't show up in the weekly
summary table. The total should be $9,500 and it still shows $8,000. And
there is no longer a 2023 trend line (it's actually 0 and I just can't see
it).  It use to say $8,000.
